data/sql/updates/pending_db_world/rev_1680443009841196300.sql

```diff
@@ -0,0 +1,39 @@
+-- Ferra loot update
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 14308) AND (`Item` IN (4500, 5759, 7909, 7910, 8146, 11414, 11415, 24016, 24018, 24020, 24033));
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 4500, 0, 0.125, 0, 1, 0, 1, 1, 'Ferra - Thorium Lockbox');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 5759, 0, 1, 0, 1, 0, 1, 1, 'Ferra - Traveler\'s Backpack');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 8146, 0, 5, 0, 1, 0, 1, 1, 'Ferra - Wicked Claw');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 11414, 0, 65, 0, 1, 0, 1, 1, 'Ferra - Grizzled Mane');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14308, 11415, 0, 27, 0, 1, 0, 1, 1, 'Ferra - Mixed Berries');
+
+-- Gems
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 7909, 0, 1, 0, 1, 1, 1, 1, 'Ferra - Aquamarine');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 7910, 0, 0.5, 0, 1, 1, 1, 1, 'Ferra - Star Ruby');
+
+-- BoEs
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 24016, 24016, 2, 0, 1, 2, 1, 1, 'Ferra - (ReferenceTable)');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 24018, 24018, 1.5, 0, 1, 2, 1, 1, 'Ferra - (ReferenceTable)');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 24020, 24020, 1, 0, 1, 2, 1, 1, 'Ferra - (ReferenceTable)');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(14308, 24033, 24033, 0.75, 0, 1, 2, 1, 1, 'Ferra - (ReferenceTable)');
```

Comment:
This can be all a single query :p 
https://www.azerothcore.org/wiki/sql-standards#compact-queries